sdcm/send_email.py

```diff
@@ -629,8 +629,9 @@ def read_email_data_from_file(filename):
     email_data = None
     if os.path.exists(filename):
         try:
-            with open(filename, "r") as fp:  # pylint: disable=invalid-name
-                email_data = json.load(fp)  # pylint: disable=invalid-name
+            with open(filename, "r") as file:
+                data = file.read().strip()
+                email_data = json.loads(data or '{}')
```

Comment:
Why code was changed? is it need to read file content to variable and then convert string to json?

Reply:
it happened couple of times, when trying to read a file (as a test can have multiple tests in it - not the same as the parallel test) and because of the content of the file was empty (file existed), then `json` library failed to `load` a `None` object... this is a way to fix it, and if there is no data in the file, we return `{}`, that will give us an empty dictionary